persistenceagent/rockcraft.yaml

```diff
@@ -16,18 +18,26 @@ services:
       NAMESPACE: ""
       TTL_SECONDS_AFTER_WORKFLOW_FINISH: 86400
       NUM_WORKERS: 2
-    command: bash -c 'persistence_agent --logtostderr=true --namespace=$NAMESPACE --ttlSecondsAfterWorkflowFinish=$TTL_SECONDS_AFTER_WORKFLOW_FINISH --numWorker $NUM_WORKERS'
-platforms:
-  amd64:
+      LOG_LEVEL: "info"
+      EXECUTIONTYPE: "Workflow"
+    command: bash -c 'persistence_agent --logtostderr=true --namespace=$NAMESPACE --ttlSecondsAfterWorkflowFinish=$TTL_SECONDS_AFTER_WORKFLOW_FINISH --numWorker $NUM_WORKERS --executionType ${EXECUTIONTYPE} --logLevel=${LOG_LEVEL}'
```

Comment:
We have to make sure that `EXECUTIONTYPE` and `LOG_LEVEL` env variables are correctly passed to the pebble component so this command is executed with them.